### ISSUE TITLE
Dangers and obstacles

### ISSUE THREAD

**Issue by Noitarud** [osmandapp/OsmAnd#20966]
### Describe the idea

I don't want to call these favourites but seems to be the way to add them. Application needs images with typical danger sign themes (lasers, radioactive, smoke) the public may see, maybe include nautical themes too.

maybe allow a do not rout near option, and warning when the rout or destination is going near one. This might combine with #7669.

### Expected behaviour

When avoiding, specify a length or radius

### Alternatives you've considered

#7669 but needs descriptive emblems

**Comment by vshcherb** [comment]
yes I got the idea, similar to road shields where you could put text in frame. Well something to think of in the future

**Comment by dmpr0** [comment]
<table style="width:100%">
  <tr>
    <td><img src="https://github.com/user-attachments/assets/306e65cc-f39f-4207-bd7e-6e6411c1201a"  width="300" height="auto"/> </td>
  </tr>
</table>   

Hazard icons are now present as an icon group for favourites and track waypoints.
